test/spec/modules/openxBidAdapter_spec.js

```diff
@@ -1142,6 +1142,37 @@ describe('OpenxAdapter', function () {
           expect(request[0].data.lre).to.equal('00000000-aaaa-1111-bbbb-222222222222');
         });
       });
+
+      describe('with the criteo id for exchanges', function () {
+        it('should not send a criteoid query param when there is no userId.criteoId defined in the bid requests', function () {
+          const request = spec.buildRequests(bidRequestsWithMediaTypes, mockBidderRequest);
+          expect(request[0].data).to.not.have.any.keys('criteoid');
+        });
+
+        it('should send a criteoid query param when userId.criteoId is defined in the bid requests', function () {
+          const bidRequestsWithLiveRampEnvelope = [{
```

Comment:
Nitpick: Not required but the naming is off on the variable

Reply:
I agree with @jimee02 here - there's no liveramp in bidRequestsWithLiveRampEnvelope

Reply:
You are right. Thanks @bretg for the fix